src/main/java/org/terasology/las/LaSWorldGenerator.java

```diff
@@ -78,13 +60,12 @@
 import org.terasology.registry.CoreRegistry;
 import org.terasology.registry.In;
 import org.terasology.world.block.BlockManager;
-import org.terasology.world.generation.BaseFacetedWorldGenerator;
 import org.terasology.world.generation.World;
 import org.terasology.world.generation.WorldBuilder;
 import org.terasology.world.generator.RegisterWorldGenerator;
 import org.terasology.world.generator.plugin.WorldGeneratorPluginLibrary;
 
-import org.terasology.las.LaSFloraRasterizer;
+//import org.terasology.core.world.generator.facetProviders.EnsureSpawnableChunkZeroProvider;
```

Comment:
don't commit commented code - it should either be in the history anyways (and thus can be restored) or is not needed at all (then why commit it).